quiz.js

```diff
@@ -1,27 +1,39 @@
-var Carlot = (function() {
-    var carInventory = [];
-
-    return {
-        getCarInventory: function(callback) {
-            // create an xhr to load carInventory
-            var xhr = new XMLHttpRequest();
-
-// Listen for when the load event is broadcast
-// and execute an anonymous callback.
-            xhr.open("GET", 'inventory.json');
-            xhr.addEventListener("load", function(event) {
-                // Set the value of the private array
-                // List the cars in the DOM
-                // Parse JSON objects into a native JavaScript Object
-                carInventory = JSON.parse(event.target.responseText).cars;
-                // carInventory = JSON.parse(this.responseText);
-
-                console.log(carInventory);
-                callback(carInventory);
-            });
-            xhr.send();
-        }
-    }
-}) ();
-
-// Carlot.getCarInventory();
\ No newline at end of file
+
+
+
+function populatePage(carInventory) {
+  console.log('2', carInventory)
+var populate = document.querySelector('#output');
+      carInventory.forEach(function (car, index) {
+        populate.innerHTML +=
+
+      `
+      <div class = 'col-md-4 carCard'>
```

Comment:
When you are creating these cards, each set of three should be in a bootstrap row.